src/Command/HistoryCommand.php

```diff
@@ -140,11 +140,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
         } else {
             $type = $input->getOption('no-numbers') ? 0 : ShellOutput::NUMBER_LINES;
             if (!$highlighted) {
-                $type = $type | ShellOutput::OUTPUT_RAW;
+                $type = $type | OutputInterface::OUTPUT_RAW;
```

Comment:
Will this work on Symfony 3?

Reply:
Yes, the OutputInterface is present since at least 2.0 : https://github.com/symfony/console/commit/44124245351cbf722f051a3d4f04a9640ebb07a3